packages/ui/components/editor/Editor.tsx

```diff
@@ -84,13 +85,13 @@ export const Editor = (props: TextEditorProps) => {
           />
           <div
             className={classNames("editor-inner scroll-bar", !editable && "!bg-subtle")}
-            style={{ height: props.height }}>
+            style={{ height: props.height, overflow: "hidden auto" }}>
```

Comment:
Why this changes?

Reply:
![Screenshot 2024-08-16 101322](https://github.com/user-attachments/assets/6f41ad80-1930-4070-9f11-ec498b1819f3)
to remove the visibility of scroll bar in x direction. as it is not needed